### PR TITLE
reorganise templates, basic detail page, alerts, basic list page

### LINKED ISSUE #2: Handle form valid response (i.e. make map)
At the moment when you submit it just fails, because there's not response sent.

### PR DESCRIPTION
**Pull request by stringfellow**
resolves #2

### CODE REVIEW

**Comment by stringfellow** [comment]
Got a bit carried away trying to make the detail page useful without putting too much effort into laying it out... it works OK, but is fugly and not overly useful. You can now submit a map successfully though!

File: maps/models.py [hunk 1]
```diff
@@ -73,10 +75,13 @@ class Event(models.Model):
             message="That doesn't look like a valid GLIDE number."
         )]
     )
+    def __unicode__(self):
+         return("{0}, {1}".format(self.get_event_type_display(),
```
Comment (stringfellow) [inline maps/models.py]: this would be much lovelier if it was PEP8 compliant :)

Comment (stringfellow) [inline maps/models.py]: Basically:

```
return("{0}, {1}".format(
    self.get_event_type_display(), self.glide_number
))
```

**Comment by stringfellow** [comment]
:+1:

**Comment by geojayuu** [comment]
:+1:

**Pull request merged**